drivers/volume/portworx/portworx.go

```diff
@@ -80,8 +87,28 @@ const (
 	cloudSnapshotInitialDelay = 5 * time.Second
 	cloudSnapshotFactor       = 1
 	cloudSnapshotSteps        = math.MaxInt32
+
+	// constants
+	cmdExecutorImage         = "openstorage/cmdexecutor:latest"
+	storkServiceAccount      = "stork-account"
+	perPodCommandExecTimeout = 900 // 15 minutes
+
+	defaultRetryInterval = 5 * time.Second
+	defaultRetryFactor   = 1
+	defaultMaxRetrySteps = 12
+	execPodStepLow       = defaultMaxRetrySteps
+	execPodStepMed       = defaultMaxRetrySteps * 3
+	execPodStepsHigh     = math.MaxInt32
+
+	podsWithRunningCommandsKey = "stork/pods-with-running-cmds"
 )
 
+// CommandTask tracks pods where commands for a taskID might still be running
+type CommandTask struct {
+	TaskID string   `json:"taskID"`
+	Pods   []string `json:"pods"`
```

Comment:
Define another type which has name and namespace and use that here to avoid parsing the name

Reply:
done